mobile/src/main/res/values/strings.xml

```diff
@@ -310,6 +310,7 @@ Mana pool will also be cleared."</string>
 <string name="wishlist_type_price">"Price"</string>
 <string name="wishlist_ascending">"Ascending"</string>
 <string name="wishlist_sort_by">"Sort By"</string>
+<string name="wishlist_dont_sort">"Don't Sort"</string>
```

Comment:
Actually, you are sorting by "Order Added", so maybe it would be better to add that option to the various "Sort By" options.